proceeding/gammapy-icrc2017.tex

```diff
@@ -91,7 +91,7 @@ \section{Introduction}
 \label{sec:intro}
 
 % CTA science tools
-The Cherenkov Telescope Array (CTA) will observe the sky in very-high-energy
+The Cherenkov Telescope Array (CTA) will observe the sky in very high energy (VHE, E > 100$\,$GeV)
```

Comment:
I think "very-high-energy" is like "gamma-ray", i.e. use dashes if used as an adjective like here, no?